source/components/org.apache.stratos.load.balancer/src/main/java/org/apache/stratos/load/balancer/endpoint/TenantAwareLoadBalanceEndpoint.java

```diff
@@ -141,6 +154,21 @@ public void send(MessageContext synCtx) {
         }
     }
 
+    private boolean currentMemberFoundInTopology(org.apache.axis2.clustering.Member currentMember) {
+        String clusterId = currentMember.getProperties().getProperty(Constants.CLUSTER_ID);
+        String memberId = currentMember.getProperties().getProperty(Constants.MEMBER_ID);
+        if ((StringUtils.isNotBlank(clusterId)) && (StringUtils.isNotBlank(memberId))) {
```

Comment:
If using Java 1.7, maybe we can use "String.isEmpty" in place of "StringUtils.isNotBlank"?